leaf-ui/rappid-extensions/LinkInspector.js

```diff
@@ -213,6 +222,10 @@ var LinkInspector = Backbone.View.extend({
     updateEndEvolRelations: function() {
         // Save based on evolving relations
         this.setValues(this.link.get('linkType'), $('#link-type-end').val(), true)
+
+        if(this.link.get('postType')!=null){
+            $('#link-type-begin option[value= \'' + this.link.get('postType') + '\']').hide(); // Hide the selected postType value in begin so that user can't duplicate
```

Comment:
```suggestion
            $('#link-type-begin option[value= \'' + this.link.get('postType') + '\']').hide(); // Hide the selected postType value in begin dropdown so that user can't select the same value for the linkType and postType
```